### DISCUSSION TITLE
Thanks for this package

### DISCUSSION THREAD

**Discussion by thefrana** [kiliman/remix-express-vite-plugin#33] [General]
Hi,
I always wanted something like this. Big thanks to @kiliman for creating and releasing this package!

Do you think it is possible to also support other servers like Fastify or Koa?

Thanks!

**Comment by kiliman** [comment]
Thanks for the lovely comments.

Yeah, that was the plan to support additional servers. Work has been hectic, so I have only had a little spare time. If anything, I'll wait until React Router 7 is released to see if there will be any breaking changes.

In the meantime, PRs are welcome.

**Comment by superjose** [comment]
Wow. I was about to create a similar post. 

@kiliman , I wanted to thank you for this package as well personally 🙏 🙏 🙏 . 

I've been painfully migrating Remix/Vite to a [turborepo monorepo with Cloudflare Workers](https://hackernoon.com/how-to-create-a-monorepo-with-vite-cloudflare-remix-pnpm-and-turborepo-no-build-step). Unfortunately, the pain of using wrangler with incompatible dependencies drove me to the brim.

The challenge was to migrate the `getLoadContext` function. I use the context as a decoupling mechanism between the remix and node layers. This way, I can replace the Backend-For-Frontend or Frontend app without affecting my code. 

The challenge was doing this with Vite and Remix as I'm using special packages like [lingui](https://www.npmjs.com/package/@lingui/core) for i18n.

I began searching for alternatives as I knew this wasn't shown in the Remix docs, and that's when I found your comment in [Remix + Vite + Express - no root js](https://github.com/remix-run/remix/discussions/8978#discussioncomment-9014176). 

Thanks for this!!!

This must be made available officially and included in the site!
